fighters/cloud/src/opff.rs

```diff
@@ -1,6 +1,7 @@
 use super::*;
+use globals::*;
 
-utils::import_noreturn!(common::opff::fighter_common_opff);
+utils::import_noreturn!(common::opff::{fighter_common_opff, b_reverse});
```

Comment:
all of these import_noreturn statements could easily be made as a fighter ext trait instead, btw. Then you wouldnt have to rename it everywhere. I would just make sure to call it something like `check_b_reverse` rather than just `b_reverse`. Just an idea.